src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -134,15 +128,28 @@ describe('HmctsGlobalHeaderComponent', () => {
 
   it('should not display find case right aligned', async () => {
     component.showItems = true;
-    component.showFindCase = false;
-    component.items = navItemsWithFindCaseRightAligned;
+    component.items = [{
+      text: 'Find case',
+      href: '/cases/case-search',
+      active: false
+    },
+    {
+      text: '2',
+      href: '',
+      active: false
+    },
+    {
+      text: '3',
+      href: '',
+      active: false
+    }];
     await component.ngOnChanges(changesMock);
     fixture.detectChanges();
     const findCase = fixture.debugElement.query(By.css('.hmcts-search-toggle__button'));
     expect(findCase).toBeFalsy();
   });
 
-  it('splitNavItems', (done: DoneFn) => {
+  it('splitNavItems', async () => {
```

Comment:
@johnbenjamin-hmcts - was it intended to move this back to `async()`? I thought @ronaldmansveld changed it from `async()` to `(done: DoneFn)`...

Reply:
Thank you @Daniel-Lam - I have now reverted the changes.